### DISCUSSION TITLE
Welcome to dayz-server-fake-players Discussions!

### DISCUSSION THREAD

**Discussion by DataGoblin** [DataGoblin/dayz-server-fake-players#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by murilo05** [comment]
To it works, i just need to put the file in the main server folder and start the server? @DataGoblin

**Reply by DataGoblin** [reply]
@murilo05 Sadly not as simple as that! This a memory exploit so it would require injecting the op code via an application such as cheat engine or pymem.

**Reply by murilo05** [reply]
Ohh, ok... And do you know how to do it? Or teach me how? I can pay for it

**Reply by DataGoblin** [reply]
I might be able to make a video in the future to demonstrate the process. Or maybe expand and make a script that will help cut out the confusion.

**Reply by Brenossx** [reply]
also could not you could exemplify the process?

**Reply by Brenossx** [reply]
try to help us with at least one tip, so it's easier to inject this code

**Reply by httpss7** [reply]
I'm done this already, contact me on Discord Kastet#3304

**Reply by Brenossx** [reply]
> I'm done this already, contact me on Discord Kastet#3304

i Know you, you sell this solution Per 50 USD, BUT I NOT GO NEEDED BECAUSE  I DISCOVERED HOW TO MAKE with cheat engine, WITH THE INFORMATION THE DataGoblin

**Reply by bmongeon** [reply]
@Brenossx how?

**Comment by Brenossx** [comment]
try to help us with at least one tip, so it's easier to inject this code

**Comment by TripleC3** [comment]
Man... i don't know if you don't play dayz or what but you really fucked up the community server search for everyone... This is not what you do when you know assembly... Or atleast if you do it for fun don't upload it man... whatever
